distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
+requests, allowing federated sources to run queries as that user. This is critical because {branding}
+must make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
+used internally to represent all subjects (even unauthenticated subjects). When federating over REST,
+the assertion is simply attached as an HTTP header to the query request, and extracted by the
+federated source when the request is received. If the assertion is signed by a trusted cert, the
+federated source will then run the query as the user.
+
+The figure below shows a federated query between two instances of {branding} that support
+federated identity.
+
+[ditaa, federated_identity, png, ${image-width}]
+....
+                  /---------------------------\           /--------------------------\
+                  | cDEF       DDF            |           |     Federated Source     |
+/--------\        | +-----------------------+ |           | cDEF                     |
+|  User  +------->| |    Send request to    | |           | +----------------------+ |
+|  cDEF  | Search | |   Catalog Framework   | |  /------->| |   Recreate subject   | |
+\--------/        | +-----------------------+ |  | Query  | +----------------------+ |
+     ^            |             |             |  |        |             |            |
+     |            |             v             |  |        |             v            |
+     |            | +-----------------------+ |  |        | +----------------------+ |
+     |            | | Create HTTP request & | +--/        | | Run query as subject | |
+     |            | | attach SAML assertion | |           | +----------------------+ |
+     |            | +-----------------------+ |           |             |            |
+     |            |                           |           |             v            |
+     |            | +-----------------------+ |           | +----------------------+ |
+     \------------+ |     Filter results    | |<----------+ |    Filter results    | |
+                  | +-----------------------+ |  Filtered | +----------------------+ |
+                  \---------------------------/   results \--------------------------/
+....
+
+. The user submits a search to {branding}.
+. {branding} generates a catalog request and attaches the user's <<_subject,Subject>>. This
+Subject stores a SAML assertion, which is generated and signed by the
+<<_security_token_service, STS>>. The catalog request gets sent to the Catalog Framework.
```

Comment:
Order the events in lines 44-56 chronologically. If the SAML Assertion must be generated before it can be stored, then reorder:

`The Secure Token Service generates and signs a SAML Assertion and the Subject stores the Assertion.
`